src/components/OutboxMessageListItem.vue

```diff
@@ -151,7 +151,7 @@ export default {
 				failed: false,
 				sendAt: (new Date().getTime() + UNDO_DELAY) / 1000,
 			}
-			await this.outboxStore.updateMessage({ message, id: message.id })
+			await this.outboxStore.sendMessage({ id: message.id, force: true })
```

Comment:
> Fixed send now to use proper send method instead of updating the message

I think this was intentional. We set the new sendAt value with the undo delay. Then the actual sending happens.